### ISSUE TITLE
AVT 3 FormInput - VO does not announce value or increment when the up and down arrows used to input the number. 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2493]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled Tab to number input. (Focus is on the up arrow)
Press the up arrow. Voice over does not announce the value or the increment number. 
Tab to the down arrow is the same.
Tab to the Number input and the value is announced.
Enter an odd number directly into the Number field.
Shift-Control-Option to enter the stepper
Control Option Up or Down Arrow keys to change the number. 
The number is not announced correctly by VoiceOver. 

## Expected result:
When on an arrow that increments the number up or down the user should be advised that the number is incrementing by 10 as well as the number that is displayed in the number input field.
If a user inputs a number directly that is in the wrong increments they should get a warning.
If they change the number using the VO Control-Option-Up or Down arrow keys the number should be announced correctly.

**Comment by snidersd** [comment]
@dakahn This issue is missing labels: A11y & React

**Comment by abbeyhrt** [comment]
The FormItem storybook example is really just our NumberInput component wrapped in the FormItem so I changed the title to address where the problem will be fixed!

**Comment by snidersd** [comment]
@abbeyhrt There is already an open issue for the Number Input component #4185. Please be sure that both the FormItem example and the NumberInput example are fixed. Thx!

**Comment by abbeyhrt** [comment]
Changed title back to ensure that we test everywhere this problem is occurring!

**Comment by joshblack** [comment]
Duplicate of #4185, should be addressed once that one is done!
